### DISCUSSION TITLE
http://localhost:8000/blog/2023-02-20-deno-first-thoughts

### DISCUSSION THREAD

**Discussion by giscus** [420bytes/web-core#6] [Announcements]
# http://localhost:8000/blog/2023-02-20-deno-first-thoughts

420bytes | Coding Queerly | Coded Gallery | PeerPaste

http://localhost:8000/blog/2023-02-20-deno-first-thoughts

**Comment by moimikey** [comment]
Test comment.

**Comment by moimikey** [comment]
Another test comment.
